### DISCUSSION TITLE
can uvtool make a exposure time finder for elegoo Saturn 3 and mars4  9K ?

### DISCUSSION THREAD

**Discussion by webtv123** [sn4k3/UVtools#898] [General]
can uvtool make a exposure time finder for elegoo Saturn 3 and mars4  9K ? these two printers are Not  (ULTRA)

**Comment by sn4k3** [comment]
It should, but you must test to really know.

**Comment by Vicarium** [comment]
Just chiming in for future users: The exposure time finder works just fine on the Mars 4 9k. 

Its been saving me a ton of time when trying new resins. Big thanks to the developers.
